components/has/overlays/staging/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/argocd-roles
+  - ../../base/external-secrets
```

Comment:
these resources are in both staging and prod, so I would think it would be nice to see them in base

Reply:
The reason why they're not in base is because `external-secrets` is not in dev